htdocs/core/tpl/objectline_edit.tpl.php

```diff
@@ -303,6 +303,76 @@
 
 <script type="text/javascript">
 
+<?php
+if (! empty($usemargins) && $user->rights->margins->creer)
+{
+?>
+	/* Some js test when we click on button "Add" */
+	jQuery(document).ready(function() {
+		<?php
+		if (! empty($conf->global->DISPLAY_MARGIN_RATES)) { ?>
+		$("input[name='np_marginRate']:first").blur(function(e) {
```

Comment:
When we edit the price, we use the "keyup" function to set or empty the rate.
So i think when we edit the rate, we should also use "keyup" instead of "blur" to set the unit price ? Is there any reason to use "blur" compared to "keyup" ?

Reply:
In add form, there is the same part of code and "keyup" event is called more times than "blur" event.